File: week-02/day-01/draw_triangle.py
# Write a program that reads a number from the standard input, then draws a
# triangle like this:
#
# *
# **
# ***
# ****
#
# The triangle should have as many lines as the number was

triangle_levels = int(input('Levels of the triangle: '))

star = '*'
for i in range(0 , triangle_levels):
    print(i * star)

File: week-07/day-01/draw_triangle.js
'use strict';

var lineCount = 4;

// Write a program that draws a
// triangle like this:
//
// *
// **
// ***
// ****
//
// The triangle should have as many lines as lineCount is

let star = '*'

for (let i = 0; i < lineCount; i += 1){
    console.log(star.repeat(i+1))
}